R/sim_anneal_helpers.R

```diff
@@ -144,11 +144,31 @@ random_real_neighbor <- function(current, hist_values, pset, retain = 1,
 
 encode_set_backwards <- function(x, pset, ...) {
   pset <- pset[pset$id %in% names(x), ]
-  new_vals <- purrr::map2(pset$object, x, dials::encode_unit, direction = "backward")
+  new_vals <- mapply(encode_unit_backwards, pset$object, x,
+                     SIMPLIFY = FALSE, USE.NAMES = FALSE)
   names(new_vals) <- names(x)
   tibble::as_tibble(new_vals)
 }
 
+encode_unit_backwards <- function(x, value) {
+  if (!dials::has_unknowns(x)) {
+    compl <- value[!is.na(value)]
+    if (any(compl < 0) | any(compl > 1)) {
```

Comment:
Let's use this logical to tell them which parameter is triggering the error.

Reply:
Ah, good thought. [245c6ff](https://github.com/tidymodels/finetune/pull/97/commits/245c6fff54a84f9b18d1b973d4eee7e933845b58)